test/integration/targets/copy/tasks/tests.yml

```diff
@@ -2448,3 +2448,44 @@
       loop:
         - '{{ src }}'
         - '{{ src }}_dest'
+
+- name: Record the output directory
+  set_fact:
+    remote_file: foo.txt
```

Comment:
A better way to do this a `block` with `vars:`

Reply:
Updated